### PR TITLE
Add scriptMessageHandler property to ORKWebViewStepViewController

### PR DESCRIPTION
**Pull request by overcyn**
Add "GetAccessToken" scriptMessageHandler and expose it on ORKWebViewStepViewController.

### CODE REVIEW

File: ORK1Kit/ORK1Kit/Common/ORK1WebViewStepViewController.h [hunk 1]
```diff
@@ -50,7 +50,8 @@ ORK1_CLASS_AVAILABLE
  a web view step in a task, and present a task view controller for that task.
  */
 @interface ORK1WebViewStepViewController : ORK1StepViewController<WKScriptMessageHandler, WKNavigationDelegate>
-
+@property (nonatomic, strong, readonly) WKWebView *webView;
```
Comment (eschramm) [inline ORK1Kit/ORK1Kit/Common/ORK1WebViewStepViewController.h:53]: Minor - if we're making this a @property, should we remove the `*_webView` declaration in the .m file on line 90?

File: ResearchKit/Common/ORKWebViewStepViewController.h [hunk 2]
```diff
@@ -48,7 +48,8 @@ NS_ASSUME_NONNULL_BEGIN
  a web view step in a task, and present a task view controller for that task.
  */
 @interface ORKWebViewStepViewController : ORKStepViewController<WKScriptMessageHandler, WKNavigationDelegate>
-
+@property (nonatomic, strong, readonly) WKWebView *webView;
```
Comment (eschramm) [inline ResearchKit/Common/ORKWebViewStepViewController.h:51]: Same as above regarding _webView ivar and @property declaration.

**Review by eschramm — COMMENTED** [review]
Minor clarity regarding @property and existing local ivar - otherwise LGTM

File: ORK1Kit/ORK1Kit/Common/ORK1WebViewStepViewController.h (hunk 1 above)
Comment (overcyn) [reply ORK1Kit/ORK1Kit/Common/ORK1WebViewStepViewController.h:53]: Updated.

File: ResearchKit/Common/ORKWebViewStepViewController.h (hunk 2 above)
Comment (overcyn) [reply ResearchKit/Common/ORKWebViewStepViewController.h:51]: Updated

**Comment by eschramm** [comment]
🚢

**Pull request merged**